web/client/utils/FeatureGridUtils.js

```diff
@@ -35,13 +35,13 @@ const getRow = (i, rows) => rows[i];
 
 /* eslint-disable */
 
-const toChangesMap = (changesArray) => changesArray.reduce((changes, c) => ({
+const toChangesMap = (changesArray) => isArray(changesArray) ? changesArray.reduce((changes, c) => ({
```

Comment:
what's the reason of this change?

Reply:
because moving the reducer default state of featuregrid caused that changesArray to passed as empty object initially, this makes the map loading forever

Reply:
You may use a default = [] that seems to be more clear. 
Did you changed the initial state? It should be anyway consistent.